pkgs/by-name/st/structurizr-cli/package.nix

```diff
@@ -24,9 +24,11 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   patches = [
-    (fetchpatch2 {
-      url = "https://patch-diff.githubusercontent.com/raw/structurizr/cli/pull/175.patch";
-      hash = "sha256-Ut9Rma54vb9UYtvHiCf8LI9i3trlCa45/60sRyKdPnQ=";
+    # Use `gradle-application-plugin` to generate scripts and dist zip instead of in-house launch script
+    # PR at https://github.com/structurizr/cli/pull/175
+    (fetchpatch {
+      url = "https://github.com/structurizr/cli/compare/v2025.05.28...1a11940d089a8d70d6e298660c6f5db638cc8d00.patch";
```

Comment:
Can we just use one single patch instead of a comparison ?

Reply:
What's the best way of doing that? Squash the commits and use that commit? Put the patch on a separate gist? Just add the patch as a file as part of this folder?

Reply:
How about fetching each patch ?

Reply:
If it makes it easier I could squash said PR into one but it would probably not aid review clarity for upstream.